admin/src/main/resources/io/buoyant/admin/css/delegator.css

```diff
@@ -11,6 +11,10 @@
   box-shadow: none;
 }
 
+.panel-default>.panel-heading-primary, .panel>.list-group .list-group>.list-group-item-primary {
+  border-left: 10px solid green;
```

Comment:
Geez this is tough since we're hitting the limits of bootstrap. My thought is to ditch the css changes and instead set the active class on the panel heading and list group items.  We can stop using bootstrap for the delegator in a different CL. 

![screen shot 2017-02-22 at 1 21 06 pm](https://cloud.githubusercontent.com/assets/41829/23232983/25f74686-f902-11e6-9307-0791e28307e5.png)